src/intro.md

```diff
@@ -129,16 +130,16 @@ Reliability
 * Improves boot reliability by replacing lightly-tested firmware drivers with
   hardened Linux drivers
 * Proven approach for almost 20 years in military, consumer electronics, and
-  supercomputing systems – wherever reliability and performance are paramount
-* Fault Tolerance - Linux isolates processes** **(for example, when Pxeboot
-  fails catastrophically, diskboot still works afterwards)
+  supercomputers – wherever reliability and performance are paramount
+* Fault Tolerance - Linux isolates processes** **(for example, when `pxeboot`
+  fails catastrophically, `diskboot` still works afterwards)
```

Comment:
should be `localboot`; we currently do not have a `diskboot` command
that is a thing in QNX and GRUB, from a quick search

Reply:
Created https://github.com/linuxboot/book/issues/126 since I'm not sure this section was originally just focused on u-root